client/signup/steps/domains/index.jsx

```diff
@@ -60,6 +61,17 @@ const DomainsStep = React.createClass( {
 		return { products: productsList.get() };
 	},
 
+	componentWillMount: function() {
```

Comment:
Adding the A/B test code to the `domains` step here relies on the order of the steps remaining the same. It might be better if we could keep all the new logic in one place, perhaps by moving the `submitSignupStep()` call into `goToNextStep()` in `main.jsx` where we added the other bit of test code.